itou/templates/apply/includes/rdv_insertion_promo_card.html

```diff
@@ -0,0 +1,35 @@
+{% load static %}
+
+<div id="rdvi-promo" class="card mb-3 mt-4 px-4 alert-dismissible-once d-none container rounded-lg border-important bg-important-lightest">
+    <div class="row">
+        <div class="col-auto align-self-center">
+            <img src="{% static 'img/logo_rdv_insertion.svg' %}" height="90" alt="" loading="lazy" class="m-auto">
+        </div>
+        <div class="col p-0">
+            <div class="card-header pt-4">
+                <span class="badge badge-pill badge-important badge-xs">Nouveau service</span>
+                <button data-dismiss="alert" data-target="#rdvi-promo" type="button" class="close" aria-label="Fermer">
+                    <i class="ri-close-line"></i>
+                </button>
+            </div>
+            <div class="card-body pb-4">
+                <h4>RDV-Insertion simplifie les rendez-vous avec vos candidats</h4>
+                <p>
+                    Besoin d'un outil de prise de RDV par mail/sms avec gestion des rappels pour
```

Comment:
_nit_ : Le "sms" (et "mail" aussi) m'a trigger donc je suis allez voir ce qui y avais écrit sur la page de RDVI ;).
```suggestion
                    Besoin d'un outil de prise de RDV par mail et/ou SMS avec gestion des rappels pour
```